src/styles/home.scss

```diff
@@ -1,11 +1,25 @@
-
 #home {
   text-align: center;
   width: 100%;
+  color: $mainTextColor;
 }
 
 #headerLogo {
   max-width: 15vh;
 }
 
+.navigation-button {
+  align-items: center;
+  background: $buttonLinkColor !important;
+  border-radius: 10px;
+  padding: 3vh;
+  transition-duration: 0.4s;
 
+  &:focus,
+  &:hover {
+    background-color: $hoverColor !important;
+    color: $buttonLinkColor !important;
```

Comment:
Lots of `!important`s. Consider things you could do to make these more specific rather than using `!important` if you can avoid it. Just makes it that much harder to override the override if needed down the road.

Reply:
The issue that I am running into here is I am using tailwind and daisy ui which apply their own default styles to my elements and I am unsure how to fix that without using the `!important`. We could look into this during our pair programming.